src/main/java/org/radarcns/phone/OfflineProcessor.java

```diff
@@ -75,13 +91,15 @@ public void onReceive(Context context, Intent intent) {
 
     /** Start processing. */
     public void start() {
+        processBroadcast();
```

Comment:
To avoid all services starting processing immediately as the app starts, I'd like to postpone this call by not calling it directly, only via alarm manager

Reply:
What do you think about immediate execution if a debugger is attached? That would make debugging much easier.
if (Debug.isDebuggerConnected()) {
  processBroadcast();
}

Reply:
Yes, sure. As long as there is also one scheduled "immediately" through the alarmmanager.

Reply:
Done